scripts/project.py

```diff
@@ -63,16 +64,21 @@ def run_securify(self, compilation_output, securify_target_output):
         cmd = ["java", f"-Xmx{memory}G", "-jar", str(self.securify_jar),
                "-co", compilation_output,
                "-o", securify_target_output]
-        if self.pretty_output:
-            cmd += ["--pretty"]
+        if self.clargs.json:
+            cmd += ["--json"]
+        if self.clargs.verbose:
+            cmd += ["-v"]
+        if self.clargs.quiet:
+            cmd += ["-q"]
 
         try:
-            self.sec_output = subprocess.check_output(
-                cmd, stderr=subprocess.STDOUT, universal_newlines=True)
-        except subprocess.CalledProcessError as e:
-            logging.error("Error running securify.")
+            subprocess.run(cmd, check=True, stdout=sys.stdout,
+                           stderr=sys.stderr, universal_newlines=True)
```

Comment:
Why do we need to specify the `stdout` and `stderr` arguments? What if we leave the defaults?

Reply:
Yes, I didn't realise that with the default `None`, it would also work. Changing this.